library/src/main/java/com/carlosdelachica/easyrecycleradapters/adapter/CommonRecyclerAdapter.java

```diff
@@ -49,11 +49,15 @@ public void setBottomReachedCallback(AdapterCallback callback) {
 
     @Override
     public ViewHolder onCreateViewHolder(ViewGroup viewGroup, int position) {
-        ViewHolder viewHolder = inflateViewHolder(viewGroup);
+        int viewType = getItemViewType(position);
```

Comment:
Please remove unnecessary blank lines and move getItemViewType(position) call as follows:

ViewHolder viewHolder = inflateViewHolder(viewGroup, getItemViewType(position);